src/Contacts/Contact.m

```diff
@@ -19,6 +28,11 @@ - (instancetype)initWithContactWithFirstName:(NSString *)firstName
         _recordID             = record;
         _image                = image;
 
+        // Next three fields are generated on-demand on first access
+        _fullName                = nil;
```

Comment:
True. Included this to be declarative and reduce change of ambiguity for future readers/contributors, but I'm fine with removing it in the interest of brevity.